lib/Controller/CallController.php

```diff
@@ -416,8 +416,14 @@ public function leaveCall(bool $all = false): DataResponse {
 		}
 
 		if ($all && $this->participant->hasModeratorPermissions()) {
+			$result = $this->roomService->resetActiveSinceInDatabaseOnly($this->room);
+			if (!$result) {
+				// Someone else won the race condition, make sure this user disconnects directly and then return
+				// FIXME $this->participantService->changeInCall($this->room, $this->participant, Participant::FLAG_DISCONNECTED);
```

Comment:
this looks like a good idea, but it makes the user being missed in the call summary. TO be investigated

Reply:
Can no longer reproduce. Maybe it was something else :see_no_evil: